server/app.rb

```diff
@@ -0,0 +1,107 @@
+require_relative 'config/environment'
+
+PORT = 9393
+set :port, PORT
+# set :environment, :production
```

Comment:
Not necessary to change in this PR, but you might want to take a look at adding `dotenv` and loading the environment using a `RACK_ENV` variable.